query-node/mappings/bootstrap.ts

```diff
@@ -55,4 +69,22 @@ export async function bootstrapData({ store }: StoreContext): Promise<void> {
     )
   })
   await Promise.all(workers.map((w) => store.save<Worker>(w)))
+
+  const channelCategories = channelCategoriesData.map(
+    (m) =>
+      new ChannelCategory({
+        id: m.id,
+        name: m.name,
```

Comment:
```suggestion
        name: m.name,
        channels: [],
        createdInBlock: 1,
        createdAt: new Date(),
        updatedAt: new Date(),
```

Something like that would work.